### PR TITLE
TIKA-3879 - add s3 testcontainers integration test

### PR DESCRIPTION
**Pull request by nddipiazza**
# add s3 tika pipes integration tests

add integration test for s3 pipe iterator, s3 fetcher, and s3 emitter.

### CODE REVIEW

File: tika-integration-tests/tika-pipes-s3-integration-tests/src/test/java/org/apache/tika/pipes/s3/tests/S3PipeIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
+package org.apache.tika.pipes.s3.tests;
```
Comment (tballison) [inline tika-integration-tests/tika-pipes-s3-integration-tests/src/test/java/org/apache/tika/pipes/s3/tests/S3PipeIntegrationTest.java:1]: Probably need a license on this?  checkstyle should have complained!

File: tika-pipes/tika-emitters/tika-emitter-s3/src/main/java/org/apache/tika/pipes/emitter/s3/S3Emitter.java [hunk 2]
```diff
@@ -16,33 +16,19 @@
  */
 package org.apache.tika.pipes.emitter.s3;
 
-import static org.apache.tika.config.TikaConfig.mustNotBeEmpty;
```
Comment (tballison) [inline tika-pipes/tika-emitters/tika-emitter-s3/src/main/java/org/apache/tika/pipes/emitter/s3/S3Emitter.java:19]: Import order had to be fixed?

**Comment by tballison** [comment]
Couple of really small things.  Thank you so much for getting this rolling!

File: tika-pipes/tika-emitters/tika-emitter-s3/src/main/java/org/apache/tika/pipes/emitter/s3/S3Emitter.java (hunk 2 above)
Comment (nddipiazza) [reply tika-pipes/tika-emitters/tika-emitter-s3/src/main/java/org/apache/tika/pipes/emitter/s3/S3Emitter.java:19]: ah yeah. i'm getting fired up on a new laptop with new everything. so i'm setting up my checkstyle profile now

**Pull request merged**